pom.xml

```diff
@@ -141,7 +141,7 @@
 		<cdi-unit.version>4.1.0</cdi-unit.version>
 		<commons-collections.version>3.2.2</commons-collections.version>
 		<commons-collections4.version>4.4</commons-collections4.version>
-		<commons-fileupload.version>1.4</commons-fileupload.version>
+		<commons-fileupload.version>2.0-SNAPSHOT</commons-fileupload.version>
```

Comment:
We will use 2.0-SNAPSHOT until it is released.
Daily snapshots are available at https://repository.apache.org/content/repositories/snapshots/org/apache/commons/commons-fileupload2/2.0-SNAPSHOT/